### ISSUE TITLE
`m_CI` is not defined, but somehow it works?

### ISSUE THREAD

**Issue by widiba03304** [KhronosGroup/OpenXR-Tutorials#141]
Hi, I'm using the tutorial code to build my app, but I noticed something odd in the graphics API:
https://github.com/KhronosGroup/OpenXR-Tutorials/blob/e493c17ca24caf5a9fec4f7085e8a520fa211888/Common/GraphicsAPI_Vulkan.cpp#L444

It seems that `m_CI` isn't defined, yet no errors have occurred.

Could you give some explanation about this code? Thanks in advance.

**Comment by widiba03304** [comment]
NVM, I found no `#define VK_USE_PLATFORM_ANDROID_KHR` in the tutorial code, but mine has. But it still needs a fix, IMHO.

**Comment by rbessems** [comment]
Yeah that is dead code. This function is only used from the desktop self-test and the Android path is unused. I've removed it in #148
